ts/parts/Series.ts

```diff
@@ -2593,6 +2593,45 @@ H.Series = H.seriesType<Highcharts.SeriesOptions>(
          * @apioption plotOptions.series.zones.value
          */
 
+        /**
+         * When using dual or multiple color axes, this number defines which
+         * colorAxis the particular series is connected to. It refers to
+         * either the
+         * {@link #colorAxis.id|axis id}
+         * or the index of the axis in the colorAxis array, with 0 being the
+         * first. Set this option to false to prevent a series from connecting
+         * to the default color axis.
+         *
```

Comment:
Missing info that older versions used only `boolean` option.

Reply:
We should silently support it but I do not see, why we should describe for the new version old behaviour - unless the option itself gets deprecated.

Reply:
There's no silent support, both old and new one supports `boolean`. Simply in new versions more possible values were added (correct me if I'm wrong).

I think it's needed. We don't have version-specific docs online: https://api.highcharts.com/highcharts/ - only `v7.1.3-modified` is available (oops.. :D ). Users have no easy way to check online what are the supported types.

If we don't do that, we can get reports that `colorAxis: 'string'` doesn't work in v7.1 + angry users that documentation is misleading. And we end up updating docs anyway..

Reply:
I agree. In this case we should do it with a positive spin and write, what was added with v7.2.0.
> Since v7.2.0 the option can be also an axis key or an axis index instead of a boolean flag.

Reply:
... can also be an axis `id` or an ...